release_notes/ocp-4-6-release-notes.adoc

```diff
@@ -60,6 +60,63 @@ This release adds improvements related to the following components and concepts.
 [id="ocp-4-6-rhcos"]
 === {op-system-first}
 
+[id="ocp-4-6-boot-live-iso-by-running-coreos-installer"]
+==== Boot live ISO by running `coreos-installer`
```

Comment:
This is actually reversed: You can use the Live ISO to get to an live environment on the system in which you can run coreos-installer to permanently install the system.

Initial text was:
It is now possible to boot an RHCOS "live ISO" into a live environment where you can run `coreos-installer`.